cmd/cadvisor.go

```diff
@@ -153,6 +160,9 @@ func main() {
 		klog.Fatalf("Failed to register HTTP handlers: %v", err)
 	}
 
+	// Generate tls.Config if it was requested by flags
```

Comment:
Comment is inaccurate. Implies the existence of a conditional that isn't there. Instead, consider something like:
`createTLSConfig returns nil if arguments are empty strings`